src/components/date_picker/super_date_picker/quick_select_popover/recently_used.tsx

```diff
@@ -59,18 +64,29 @@ export const EuiRecentlyUsed: FunctionComponent<EuiRecentlyUsedProps> = ({
   });
 
   return (
-    <Fragment>
+    <fieldset>
       <EuiTitle size="xxxs">
-        <span>Recently used date ranges</span>
+      <legend id={legendId} aria-label="Recently used date ranges">
```

Comment:
The `aria-label` is the same as the inner text so we don't need one here. (Also the inner text is internationalized so that's better too.)

```suggestion
      <legend id={legendId}>
```